cmd/opm/alpha/generate/cmd.go

```diff
@@ -52,6 +52,7 @@ value of each duplicate key will be added to the generated Dockerfile.
 			}
 
 			dir, indexName := filepath.Split(fromDir)
+			indexName = filepath.Clean(indexName)
```

Comment:
Does it also work to `filepath.Clean(fromDir)` at the very beginning. It seems like that could help us avoid future regressions if we later make other manipulations based on `fromDir`.